bigbluebutton-html5/imports/ui/components/whiteboard/cursors/component.jsx

```diff
@@ -173,8 +176,9 @@ export default function Cursors(props) {
   const handleGrabbing = () => setPanGrabbing(true);
   const handleReleaseGrab = () => setPanGrabbing(false);
 
+  const multiUserAccess = hasMultiUserAccess(whiteboardId, currentUser?.userId);
```

Comment:
this line is triggering an error: `imports/ui/components/whiteboard/cursors/component.jsx:349:8: /home/bigbluebutton/src/bigbluebutton-html5/imports/ui/components/whiteboard/cursors/component.jsx: Identifier 'multiUserAccess' has already been declared. (349:8)`

Reply:
Ops, fixed